Std/Data/List/Basic.lean

```diff
@@ -424,6 +424,8 @@ will be retained (but order will otherwise be preserved).
 
 instance [DecidableEq α] : Union (List α) := ⟨List.union⟩
 
+theorem union_def [DecidableEq α] (l₁ l₂ : List α)  : l₁ ∪ l₂ = foldr .insert l₂ l₁ := rfl
```

Comment:
Move it to `Lemmas`, `Basic` is only for definitions and lemmas required for definitions

Reply:
Done.